.github/workflows/apply-library-repo-templates.yml

```diff
@@ -0,0 +1,77 @@
+name: Apply Library Repo Templates and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      template_type:
+        description: 'Template type'
+        default: 'Generated Connector Template (Standard)'
+        type: choice
+        options:
+          - Generated Connector Template (Standard)
```

Comment:
We have a single option here?

Reply:
Kept it intentionally to emphasize that the same workflow will be used for other library template types as well. But if you prefer we can remove it too. wdyt?

Reply:
+1 to keep this.